src/main/kotlin/com/zopa/ktor/opentracing/Coroutines.kt

```diff
@@ -14,11 +14,13 @@ import kotlin.coroutines.EmptyCoroutineContext
 
 
 fun tracingContext(): CoroutineContext {
-    val activeSpan: Span? = threadLocalSpanStack.get()?.peek()
+    val activeSpan: Span? = getGlobalTracer().scopeManager().activeSpan()
+
```

Comment:
Re-use the read from the thread local in the Scope manager implementation here; https://github.com/zopaUK/ktor-opentracing/blob/7709211b5716bd50801d710c517258f3f00c452c/src/main/kotlin/com/zopa/ktor/opentracing/ThreadContextElementScopeManager.kt#L23